Makefile

```
@@ -134,3 +134,9 @@ fuzzing:
 	@echo "Running fuzzing tests"
 	go clean -testcache
 	go test -tags "$(TAGS)" ./... -run=^Fuzz -v -p 1
+
+.PHONY: codespell
+codespell:
+	@echo "Running codespell"
+	pip3 install codespell
```

Could you make this line as a comment? We don't install any program in the Makefile (gofmt, golangci-lint). So let it be the same.
```suggestion
	# pip3 install codespell
```

---

May be it is worth to move the installation to deps target?

---

Fixed.